vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -166,8 +165,7 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup>
     <Midl>
-      <!-- Custom XAML types must be in the project namespace to be usable by XAML -->
-      <PreprocessorDefinitions>PROJECT_ROOT_NAMESPACE=$(RootNamespace);%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
><PreprocessorDefinitions>%(PreprocessorDefinitions)</PreprocessorDefinitions> [](http://example.com/codeflow?start=6&length=77)

It can be removed completely then.